server/server.go

```diff
@@ -248,7 +248,6 @@ func (p *proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		io.Copy(w, resp.Body)
 		return
 	}
-	log.Printf("Response for %q received after %s", id, time.Since(pending.startTime))
```

Comment:
Can we defer this instead of removing it?

I.E. move it up to ~line 233 and invoke it using `defer log.Printf(...)`?

Reply:
Ack